scripts/generate_sheet.py

```diff
@@ -18,6 +20,7 @@ def generate_sheet(model_name, token, days, threshold):
 
     model_class = get_model_class(model_name)
     model = model_class.load(model_file_name)
+    _, all_labels = model.get_labels()
```

Comment:
I think we can use `model.class_names` here as class_names are already sorted, this way we don't need to sort them in the models themselves.